proposals/0054-sysvar-get-active-stake.md

```diff
@@ -0,0 +1,89 @@
+---
+simd: '0054'
+title: Sysvar for active stake
+authors:
+  - x19
+category: Standard
+type: Core
+status: Draft
+created: 2023-06-7
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+We propose to add a new sysvar that contains vote account pubkeys and 
+their corresponding active stake.
+
+## Motivation
+
+Currently, if a validator wants to prove its active stake to a program, 
+it needs to 
+pass in all of the stake accounts which have delegated to it. This is 
```

Comment:
nit: there are a few truncated line breaks like this through out... would be good to clean them up